Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -1,8 +1,36 @@
+var isSelectionExportable = function (artboards) {
+    var artboardPixelLimit = 1000000000; // 1e9
+    var overSizedArtboardName = nil;
+
+    var artboardsLoop = [artboards objectEnumerator];
+    var artboard = nil;
+    while (artboard = [artboardsLoop nextObject]) {
+        var artboardSize = artboard.rect().size;
+        if (artboardSize.width * artboardSize.height > artboardPixelLimit) {
+            overSizedArtboardName = artboard.name();
+            break;
+        }
+    }
+
+    artboardSize = nil;
+    artboard = nil;
+    artboardsLoop = nil;
+    artboardPixelLimit = nil;
+
+    if (overSizedArtboardName) {
+        [NSApp displayDialog:@"Selected artboard “" + overSizedArtboardName + "” is too large to export. Due to a limitation in Sketch, it’s only possible to export artboards smaller than 30,000 px ⨉ 30,000 px.\n\n☝️ You can divide the artboard into multiple artboards and export again." withTitle:@"Artboard too large"];
```

Comment:
We check for total number of pixels (=30000*30000), but this message makes me a bit confused. Are 15000 x 60000 artboards ok? If so should we incorporate the total allowed number of pixels to the message somehow? e.g. `...export artboards with less than 1 megapixels(i.e 30,000 px x 30,000 px)...`

Reply:
Yeap, theoretically all artboards less than `1e9` pixels are welcomed. We (me and @berkcebi) also discuss this and decided to display as simple warning as possible to avoid confusion. 

I think saying something like `1 megapixels ...` would not confuse the designers on export process. wdyt Berk?

Reply:
I recommend going with simplicity over accuracy. For a designer, 1 megapixel would be difficult to understand. By providing width and height limits separately, we're making the error message simpler (even though it's not 100% accurate).